libs/api/domains/translations/src/lib/translations.service.ts

```diff
@@ -23,19 +30,25 @@ export class TranslationsService {
     namespaces?: string[],
     lang?: string,
   ): Promise<TranslationsDict | null> => {
+    const locale = locales.find((l) => l.code === localeMap[lang])
+
     const result = await this.contentfulRepository
-      .getLocalizedEntries<any>(lang, {
+      .getLocalizedEntries<any>('*', {
```

Comment:
so this fetches everything?

Reply:
yes this fetches the field in all locales